e2e-tests/wiremock/src/main/java/com/foo/wiremock/WireMockApplication.java

```diff
@@ -0,0 +1,18 @@
+package com.foo.wiremock;
+
+
+import static org.hamcrest.Matchers.is;
+import static io.restassured.RestAssured.given;
+
+public class WireMockApplication {
+
+    public void callApi(int port) {
```

Comment:
add a bit description about this endpoint, eg, call another service Y which could be mocked in the manual test.
In addition, instead of asserting the call, should the endpoint return some info (eg, `Working`) which is received from the call?